### PR TITLE
fix spelling

### PR DESCRIPTION
**Pull request by kavyab2006**

### CODE REVIEW

File: .env-example
Comment (josephmaxwellhyland) [inline .env-example:1]: We actually want to keep this file around!

**Review by josephmaxwellhyland — CHANGES_REQUESTED** [review]

File: .env-example
Comment (jvgusch) [reply .env-example:1]: It seems like only whitespace changes were made to this file. Not sure how that happened though, maybe a difference in line-ending settings?

Comment (josephmaxwellhyland) [reply .env-example:1]: hmmm yeah... they look identical to me

**Review by josephmaxwellhyland — APPROVED** [review]
Weird stuff with the diff but it looks good!

File: .env-example
Comment (josephmaxwellhyland) [reply .env-example:1]: I think I'm going to say it's good

**Pull request merged**